src/pages/HelpPage.vue

```diff
@@ -17,22 +19,22 @@
         <card>
           <h2>Attacks Performed On Your Model</h2><hr>
           <p style="line-height:20px">After a thorough literature review of the attack methods of the Cleverhans library, the following attack methods have been chosen to check the robustness of your model.</p>
-          <p style="text-indent:2em">＊ FastGradientMethod</p>
-          <p style="text-indent:2em">＊ BasicIterativeMethod</p>
-          <p style="text-indent:2em">＊ MomentumIterativeMethod</p>
-          <p style="text-indent:2em">＊ CarliniWagnerL2</p>
-          <p style="text-indent:2em">＊ MadryEtAl</p>
+          <p style="text-indent:2em">＊ <a href="https://arxiv.org/pdf/1412.6572.pdf"> FastGradientMethod</a></p>
```

Comment:
Maybe delete this list and move the links to the paper titles below.

Reply:
Good idea.